lib/generators/arclight/templates/catalog_controller.rb

```diff
@@ -137,9 +137,16 @@ class CatalogController < ApplicationController
     ##
     # Configuration for partials
     config.index.partials.insert(0, :index_breadcrumb)
-    
-    config.show.metadata_partials = [:summary_field, :access_field]
-    
+
+    config.show.metadata_partials = [
+      :summary_field,
+      :access_field,
+      :background_field,
+      :scope_and_arrangement_field,
+      :related_field,
+      :admin_info_field
+    ]
```

Comment:
Are you proposing moving this config to `engine.rb`? Is so, I would vote no. This is the configuration that the design calls for, but what if I wanted to add or remove something? I guess you could do it, but it would be a bit hidden to do so